src/applications/toe/helpers.jsx

```diff
@@ -245,3 +245,50 @@ export const applicantIsChildOfSponsor = formData => {
 
   return sponsor?.relationship === SPONSOR_RELATIONSHIP.CHILD;
 };
+
+export function transformTOEForm(_formConfig, form) {
+  const payload = {
+    claimant: {
+      claimantId: 0,
+      suffix: form?.data?.relativeFullName?.suffix,
+      dateOfBirth: form?.data?.relativeDateOfBirth,
+      firstName: form?.data?.relativeFullName?.first,
+      lastName: form?.data?.relativeFullName?.last,
+      middleName: form?.data?.relativeFullName?.middle,
+      notificationMethod: form?.data?.preferredContactMethod,
+      contactInfo: {
+        addressLine1: form?.data?.relativeAddress?.street,
+        addressLine2: form?.data?.relativeAddress?.street2,
+        city: form?.data?.relativeAddress?.city,
+        zipcode: form?.data?.relativeAddress?.postalCode,
+        emailAddress: form?.data['view:otherContactInfo']?.email,
+        addressType: 'DOMESTIC',
```

Comment:
This should not be hard-coded. It should be either "DOMESTIC" or "MILITARY_OVERSEAS" based on the livesOnMilitaryBase boolean.